keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -146,10 +138,24 @@ _This section must be completed when targeting alpha to a release._
     - Feature gate name: PreferNominatedNode
     - Components depending on the feature gate: kube-scheduler
 
+* **Does enabling the feature change any default behavior?**
+  Yes. The pod with the nominated node set will be evaluated first in any scheduling cycle,
```

Comment:
It's not the pod that will be evaluated first - it's its nominated node, right?

Reply:
correct.